### PR TITLE
Run migration and db seeding at root

### LINKED ISSUE #3526: Automate dev environment for parser to run normally
Refer to this comment https://github.com/Seneca-CDOT/telescope/pull/3413#pullrequestreview-947240607

With our changes to Supabase/Prisma migrate. We need to automate these steps to seed the feeds for `Parser` to work in development mode
```bash
pnpm services:start
cd src/db
cp env.example .env
pnpx prisma migrate dev
cd ../../tools/migrate
cp env.example .env
pnpm to_supabase
```

### PR DESCRIPTION
**Pull request by DukeManh**
Add `DATABASE_URL` to env.development to run migration at root.
Add data seeding of `feeds` and `quotes` for local development.

fix #3526 

## Steps to test this PR
```
cp config/env.development .env
docker-compose up -d db kong rest studio meta
pnpm db:init
```

Go to the studio at 'localhost:8910' to inspect the seeded data.

### CODE REVIEW

File: config/env.development [hunk 1]
```diff
@@ -329,3 +329,6 @@ KONG_HTTPS_PORT=8912
 
 ## DB port
 POSTGRES_PORT=8913
+
+## DB connection string
+DATABASE_URL="postgresql://postgres:your-super-secret-and-long-postgres-password@localhost/postgres"
```
Comment (humphd) [inline config/env.development:334]: You don't need the quotes

**Review by humphd — CHANGES_REQUESTED** [review]
This is great, thanks for doing this.

Also, could we eliminate the step where you have to copy the `env` file into the root?  If devs do this, they'll run into problems later, when we use that file vs. the correct `config/*` file.

Can we instead source the env file we need by preloading it with `dotenv/config`? https://github.com/motdotla/dotenv#preload

File: package.json [hunk 2]
```diff
@@ -21,7 +21,8 @@
     "clean": "pnpm turbo run clean && pnpm -r exec rm -rf node_modules",
     "prettier": "prettier --write \"./**/*.{md,jsx,json,html,css,js,yml,ts,tsx}\"",
     "prettier-check": "prettier --check \"./**/*.{md,jsx,json,html,css,js,yml,ts,tsx}\"",
-    "migrate": "pnpm migrate --prefix src/db",
+    "migrate": "prisma migrate dev --schema=src/db/prisma/schema.prisma && prisma generate --schema=src/db/prisma/schema.prisma",
```
[the remark this answers is not in the document]
Comment (DukeManh) [reply package.json:24]: sounds good.

File: package.json [hunk 3]
```diff
@@ -21,7 +21,8 @@
...
+    "seed": "prisma db seed",
```
[the remark this answers is not in the document]
Comment (DukeManh) [reply package.json:25]: sounds good

**Comment by DukeManh** [comment]
@humphd, I don't think we have much control over the env var Prisma uses https://www.prisma.io/docs/guides/development-environment/environment-variables.

**Comment by humphd** [comment]
I'm saying something different: load the environment with the `env.development` via your script vs having the user do it.

**Comment by DukeManh** [comment]
@sirinoks, could you help follow the test steps and let me know what you think

**Comment by sirinoks** [comment]
> @sirinoks, could you help follow the test steps and let me know what you think

I can't, those don't work for me

**Comment by DukeManh** [comment]
What error do you have?

**Comment by sirinoks** [comment]
image

**Comment by DukeManh** [comment]
The e2e test is expected to fail because I updated the workflows and this is running the workflows on master

**Comment by rclee91** [comment]
> I'm pretty sure I can't use docker

@sirinoks did you have docker desktop open before you ran it? (Assuming you're on windows)

**Comment by tcvan0707** [comment]
Is this what you expect @DukeManh?
```
telescope(sensible): pnpm db:seed

> @senecacdot/telescope@3.0.0-alpha.0 db:seed /Users/thanhvan/Desktop/telescope
> prisma db seed

Environment variables loaded from .env
Running seed command `node src/db/seed` ...
Seeded 821 feeds
Seeded 29 quotes

🌱  The seed command has been executed.
```

And inside `localhost:8910`
image

**Comment by DukeManh** [comment]
@tcvan0707, perfect, you can also run `db:init` that'll run migration + seeding

**Review by tcvan0707 — APPROVED** [review]

**Comment by DukeManh** [comment]
@humphd, I've tried `node -r dotenv/config src/db/seed dotenv_config_path=/config/env.development`, it does not seem to work without `.env`, let's come back later if people have problems running the `init` command.

**Review by TueeNguyen — APPROVED** [review]

**Pull request merged**